api/src/decorators/activity-log-metadata.decorator.ts

```diff
@@ -2,7 +2,9 @@ import { SetMetadata } from '@nestjs/common';
 
 export type ActivityLogMetadataType = Array<{
```

Comment:
We now have the ability to pass a value into the activity log interceptor. We are, as usual, automatically collecting the lottery status change on `lotteryStatus`. But now on `lotteryGenerate` which itself changes the status to ran, we are passing in a default status of `ran` to track in the activity log through the interceptor.